frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -109,6 +112,16 @@ export default function SurveyEdit(): JSX.Element {
     } = useActions(surveyLogic)
     const { surveysMultipleQuestionsAvailable } = useValues(surveysLogic)
     const { featureFlags } = useValues(enabledFeaturesLogic)
+    const sortedItemIds = survey.questions.map((_, idx) => idx.toString())
+
+    function onSortEnd({ oldIndex, newIndex }: { oldIndex: number; newIndex: number }): void {
+        function move(arr: SurveyQuestion[], from: number, to: number): SurveyQuestion[] {
+            const clone = [...arr]
+            Array.prototype.splice.call(clone, to, 0, Array.prototype.splice.call(clone, from, 1)[0])
+            return clone.map((child, order) => ({ ...child, order }))
```

Comment:
maybe I'm missing something here, but why are we now adding the `order` as a property to the survey question? Don't think this is in the type 👀